test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env bash
+##############################################################################################
+# Check branches to ensure forward/backward compatibility, including some pgrade/downgrade testing.
```

Comment:
Thanks for the comment update. There's a typo `pgrade` (sorry for being picky :)

Reply:
@lukech, picky is *terrific* :)
Sorry for being so sloppy. I'll fix that once I know more about `mongdb-4.4`.

Reply:
Fixed in 7ee994fa8.